scripts/test/run-tests.sh

```diff
@@ -151,30 +151,15 @@ if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "ubuntu-qemux86-64-no-grub-d" ];
                    || test_result=$?
 fi
 
-if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi3" ]; then
+if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi4_bookworm_64bit" ]; then
   MATCHED_A_TEST=1
   # For this test we test compressed image to verify xz compression
-  wget --progress=dot:giga -N ${RASPBIAN_IMAGE_URL} -P input/image/
-  RASPBIAN_IMAGE_COMPRESSED="${RASPBIAN_IMAGE_URL##*/}"
-  convert_and_test "raspberrypi3" \
+  wget --progress=dot:giga -N ${RASPIOS_IMAGE_URL} -P input/image/
+  RASPIOS_IMAGE_COMPRESSED="${RASPIOS_IMAGE_URL##*/}"
+  convert_and_test "raspberrypi4_64" \
                    "release-1" \
-                   "input/image/${RASPBIAN_IMAGE_COMPRESSED}" \
-                   "--config configs/raspberrypi3_config $EXTRA_CONFIG" \
-                   "--" \
-                   "$@" \
-                   || test_result=$?
-fi
-
-if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "ubuntu-raspberrypi3" ]; then
-  MATCHED_A_TEST=1
-  wget --progress=dot:giga -N ${UBUNTU_SERVER_RPI_IMAGE_URL} -P input/image/
```

Comment:
This one is good to fully remove. Also please remove `UBUNTU_SERVER_RPI_IMAGE_URL` declaration above

Reply:
Good call, done!